scala/play/build.sbt

```diff
@@ -7,7 +7,7 @@ lazy val root = (project.in(file("."))
   .settings(
     libraryDependencies ++= Seq(
       guice,
-      "com.typesafe.play" %% "play-json" % "[2.8,2.9)"
+      "com.typesafe.play" %% "play-json" % "[2.9,2.10)"
```

Comment:
is it possible to use something like `last.integration` ?

I mean this `PR` has no real value facing this project

Reply:
I wouldn't put `last.integration`. I don't fully trust what can break or not.
I would also skip patch versions.

Reply:
I mean is there any way to tell Renovate like "don't bug me fo x.y.z+1"?

Reply:
I think using `[2.9,2.10)` means it 😛  and it is the only way I know